clients/rospy/src/rospy/impl/transport.py

```diff
@@ -93,15 +93,16 @@ def fileno(self):
     ## closed. Function will be passed this transport as an argument.
     ## @param self
     ## @param cleanup_callback fn(Transport): callback for when connection is closed
-    def set_cleanup_callback(self, cleanup_callback):
-        self.cleanup_cb = cleanup_callback
+    def add_cleanup_callback(self, cleanup_callback):
```

Comment:
I'll make an alias, should not affect anybody's implementations unless they were banking on callback refs being overwritten.

Should I consider the cleanup_cb attribute to be public, or fix the tests to check the new list instead?